modules/server/src/graphqlRoutes.js

```diff
@@ -249,19 +251,35 @@ export const createSchemasFromConfigs = async ({
 			configsFromFiles,
 		);
 
+		const commonFields = { fieldsFromMapping, typesWithMappings };
+
 		const { mockSchema, schema } = await createSchema({
 			enableAdmin,
 			getServerSideFilter,
 			graphqlOptions,
 			types: typesWithMappings,
 		});
 
-		return {
-			fieldsFromMapping,
-			mockSchema,
-			schema,
-			typesWithMappings,
-		};
+		if (enableNetworkAggregation) {
+			const { networkSchema } = await createSchemaFromNetworkConfig(
+				configsFromFiles[ConfigProperties.NETWORK_AGGREGATION],
+			);
+			const [mergedSchema, mergedMockSchema] = mergeSchemas({
+				local: { schema, mockSchema },
+				network: { schema: mergedSchema, mockSchema: mergedMockSchema },
+			});
+			return {
+				...commonFields,
+				schema: mergedSchema,
+				mockSchema: mergedMockSchema,
+			};
+		} else {
+			return {
+				...commonFields,
+				mockSchema,
+				schema,
+			};
+		}
```

Comment:
personally preferring the readability of verbose returns instead of adding `let` variables to conditional assign values, then more inline in a return statement like: `return {schema: mergedSchema || schema}`.